README.md

```diff
@@ -153,6 +153,24 @@ curl 'https://nodes.tox.chat/json' -s | jq -r '.nodes[] | .public_key + " " + .i
 while read pk addr; do echo "  - pk: $pk"; echo "    addr: $addr"; done
 ```
 
+## Using docker
+
+tox-node image is pushed to docker hub. You can run tox-node using docker like this command format.
```

Comment:
There is a [docker image](https://hub.docker.com/r/toxrust/tox-node) of tox-node. You can run tox-node using docker like this: